### DISCUSSION TITLE
No collections displayed when adding an item

### DISCUSSION THREAD

**Discussion by AznoYT** [dani-garcia/vaultwarden#4439] [Q&A]
### Subject of the issue
When we want to add a new item to a particular collection, it displays “No collection to display.” although there are multiple collections,
we need to change the organization and then come back to it so that it displays the list of available collections

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.5
* Web-vault version: v2024.1.2b
* OS/Arch: linux/x86_64
* Running within a container: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Forwarded-For)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: false
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.44.0
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** 

```json
{
  "_duo_akey": null,
  "_enable_duo": false,
  "_enable_email_2fa": false,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_icon_service_csp": "",
  "_icon_service_url": "",
  "_ip_header_enabled": true,
  "_smtp_img_src": "cid:",
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_session_lifetime": 20,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "auth_request_purge_schedule": "30 * * * * *",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_conn_init": "",
  "database_max_conns": 10,
  "database_timeout": 30,
  "database_url": "***************",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://********************",
  "domain_origin": "*****://********************",
  "domain_path": "",
  "domain_set": true,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_attempts_limit": 3,
  "email_change_allowed": true,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 3 * * * *",
  "emergency_request_timeout_schedule": "0 7 * * * *",
  "enable_db_wal": true,
  "event_cleanup_schedule": "0 10 0 * * *",
  "events_days_retain": null,
  "experimental_client_feature_flags": "fido2-vault-credentials",
  "extended_logging": true,
  "helo_name": null,
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_expiration_hours": 120,
  "invitation_org_name": "*****",
  "invitations_allowed": true,
  "ip_header": "X-Forwarded-For",
  "job_poll_interval_ms": 30000,
  "log_file": null,
  "log_level": "Info",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "org_events_enabled": false,
  "org_groups_enabled": false,
  "password_hints_allowed": true,
  "password_iterations": 100000,
  "push_enabled": false,
  "push_identity_uri": "https://identity.bitwarden.com",
  "push_installation_id": "***",
  "push_installation_key": "***",
  "push_relay_uri": "https://push.bitwarden.com",
  "reload_templates": false,
  "require_device_email": false,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sendmail_command": null,
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": false,
  "signups_allowed": false,
  "signups_domains_whitelist": "",
  "signups_verify": true,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": "Login",
  "smtp_debug": false,
  "smtp_embed_images": true,
  "smtp_explicit_tls": null,
  "smtp_from": "**********************",
  "smtp_from_name": "Lock ******",
  "smtp_host": "************",
  "smtp_password": "***",
  "smtp_port": 465,
  "smtp_security": "force_tls",
  "smtp_ssl": null,
  "smtp_timeout": 15,
  "smtp_username": "**********************",
  "templates_folder": "data/templates",
  "tmp_folder": "data/tmp",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_sendmail": false,
  "use_syslog": false,
  "user_attachment_limit": null,
  "user_send_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": false,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```
</details>

### Steps to reproduce
We use the VaultWarden Docker container with 3 organizations to configure on it, 4 users with the 3 organizations to configure.

No specific parameters other than domains

### Expected behaviour
Normally when adding, if you have selected a collection in the left column, the identifier you want to add is automatically put in the collection.

### Actual behaviour
We have updated the VaultWarden server since the problem appeared, we tried to lower the version of the Windows applications in order to verify that the problem does not come from the applications but nothing has changed

### Troubleshooting data
![CaptureVaultWarden](https://github.com/dani-garcia/vaultwarden/assets/19228581/ce63ce32-0c17-4cbf-80b2-236dde95d038)
![VaultCollection](https://github.com/dani-garcia/vaultwarden/assets/19228581/653f9993-3900-4d29-88e8-19591de106b3)

**Comment by AleksejEgorov** [comment]
We faced same issue too.

**Comment by n1am** [comment]
I noted this too.
A simple workaround to solve the issue is to set the ownership (Who owns this item?) to the private vault and then change it back to the organization.

**Comment by BlackDex** [comment]
Which client version are you using
You mentioned lowering it, but never mentioned which versions you tested.

**Comment by BlackDex** [comment]
Also, try a manual sync and see what happens.

**Comment by AleksejEgorov** [comment]
> Which client version are you using You mentioned lowering it, but never mentioned which versions you tested.

Bitwarden Version 2024.3.0
Shell 28.2.4
Renderer 120.0.6099.291
Node 18.18.2
Architecture x64

Vaultwarden 1.30.5

Manual sync has no effect,

**Comment by BlackDex** [comment]
I get the exact same behavior with `vault.bitwarden.com`.
This is a client issue, which we can not fix.

I suggest to report this upstream at Bitwarden and also try it your self on there Cloud service.

**Comment by SamotniQ** [comment]
I have the same problem:


Server Installed 1.32.0-978f0092
Web Installed 2024.6.2b
Database SQLite: 3.46.0

Anybody solved it?
